mail-client/bower/bower-0.9.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multiprocessing
+
+DESCRIPTION="A curses terminal client for the Notmuch email system"
+HOMEPAGE="https://github.com/wangp/bower"
+SRC_URI="https://github.com/wangp/bower/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="tinfo"
+
+DEPEND="app-crypt/gpgme:=
+	>=dev-lang/mercury-11.07"
+RDEPEND="app-crypt/gpgme:=
+	net-mail/notmuch
+	sys-apps/coreutils
+	sys-libs/ncurses"
+
+QA_PRESTRIPPED="usr/bin/bower"
```

Comment:
Fix it not to strip stuff. There's no justification for stripping here.

Reply:
I addressed all issues you mentioned. Next round. 😉